### PR TITLE
Support client.add api method

### PR DESCRIPTION
**Pull request by mbelang**

### CODE REVIEW

File: fake_ubersmith/api/methods/client.py [hunk 1]
```diff
@@ -50,6 +50,18 @@ def hook_to(self, entity):
...
         )
+        entity.register_endpoints(
+            ubersmith_method='client.add',
+            function=self.client_add
+        )
+
+    def client_add(self, form_data):
+        client_id = len(self.clients)
+
+        form_data["clientid"] = client_id
```
Comment (pboutin) [inline fake_ubersmith/api/methods/client.py]: Line 67 it is `form_data["client_id"]` is the `_` missing ?

Comment (msmol) [inline fake_ubersmith/api/methods/client.py]: no, ub returns it like that

Comment (idjaw) [inline fake_ubersmith/api/methods/client.py]: yeah. UB uses clientid

Comment (idjaw) [inline fake_ubersmith/api/methods/client.py]: If we want to change client_id to clientid I think we should do it another PR since it would be a different issue category.
